docs/ruleset_functions/get_HVAC_building_area_types_and_zones.md

```diff
@@ -0,0 +1,177 @@
+	- `lighting_space_lookup["MANUFACTURING_FACILITY_EXTRA_HIGH_BAY_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["MANUFACTURING_FACILITY_HIGH_BAY_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["MANUFACTURING_FACILITY_LOW_BAY_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["MUSEUM_GENERAL_EXHIBITION_AREA"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["MUSEUM_RESTORATION_ROOM"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["POST_OFFICE_SORTING_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["RELIGIOUS_FACILITY_FELLOWSHIP_HALL"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["RELIGIOUS_FACILITY_WORSHIP_PULPIT_CHOIR_AREA"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["RETAIL_FACILITIES_DRESSING_FITTING_ROOM"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["RETAIL_FACILITIES_MALL_CONCOURSE"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["SPORTS_ARENA_PLAYING_AREA_CLASS_I_FACILITY"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["SPORTS_ARENA_PLAYING_AREA_CLASS_II_FACILITY"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["SPORTS_ARENA_PLAYING_AREA_CLASS_III_FACILITY"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["SPORTS_ARENA_PLAYING_AREA_CLASS_IV_FACILITY"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["TRANSPORTATION_FACILITY_BAGGAGE_CAROUSEL_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["TRANSPORTATION_FACILITY_AIRPORT_CONCOURSE"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["TRANSPORTATION_FACILITY_TICKET_COUNTER"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["WAREHOUSE_STORAGE_AREA_MEDIUM_TO_BULKY_PALLETIZED_ITEMS"] = "HEATED-ONLY_STORAGE"
+	- `lighting_space_lookup["WAREHOUSE_STORAGE_AREA_SMALLER_HAND_CARRIED_ITEMS"] = "HEATED-ONLY_STORAGE"
+
+- For each building segment in RMR: `for building_segment in RMR.building.building_segments:`
+	- Create list_of_zones for storing zone_ids `list_of_zones = []`
+	- For each zone in building_segment: `for zone in building_segment.zones:`
+		- add zone_id to list_of_zones: `list_of_zones.append(zone.id)`
+		- create space_area_type variable and set equal to "None": `space_area_type = "None"`
+		- check if there is an HVAC Building Area Type: `if building_segment.area_type_heating_ventilation_air_conditioning_system != "Null":`
+			- set space_area_type equal to HVAC BAT: `space_area_type = building_segment.area_type_heating_ventilation_air_conditioning_system`
+		- else:
+			- look for a lighting_building_area_type: `if building_segment.lighting_building_area_type != "Null":`
+				- set space_area_type based on the lookup table: `space_area_type = lighting_space_lookup[building_segment.lighting_building_area_type`
+		- For each space in zone: `for space in zone.spaces:`
+			- check if there is a value in space_area_type: `if space_area_type != "None":`
+				- add space area: `list_buildiung_area_types_with_total_area_and_zones[space_area_type]["AREA"] += space.floor_area`

```

Comment:
I wonder if it's an issue when zone has spaces with different lighting area type. Not sure we zound be splitting a zone into different HVAC area types.

Reply:
I agree. Since it's generally unlikely that there is more than one space per zone in the model, I would set zone area type based on the lighting area type of spaces that account for the greatest area withing a zone.

Reply:
> I agree. Since it's generally unlikely that there is more than one space per zone in the model, I would set zone area type based on the lighting area type of spaces that account for the greatest area withing a zone.

done